apps/aechannel/src/aesc_channels.erl

```diff
@@ -225,6 +226,21 @@ serialization_template(?CHANNEL_VSN) ->
     , {closes_at        , int}
     ].
 
+-spec serialize_for_client(channel()) -> map().
+serialize_for_client(#channel{} = Ch) ->
+    #{<<"id">>               => id(Ch),
+      <<"initiator">>        => initiator(Ch),
```

Comment:
I see the `channel_id`, `initiator` ,`responder`, `delegates` and `state_hash` here are the plain public keys (binaries). Shouldn't we have those as aec_base58 encoded instead?

Reply:
✅